### ISSUE TITLE
OS X application bundle support

### ISSUE THREAD

**Issue by stopiccot** [jpakkane/meson#48]
It's seems that meson has no support for .app bundles

**Comment by jpakkane** [comment]
That is true. The main reason for this is that I have never needed to do app bundles so I don't know what is the proper way to do them. I have looked into this somewhat but at the moment this is not a priority. Patches and use case explanations are welcome.

**Comment by jpakkane** [comment]
There is now a wiki page and sample code on how to get this done. Fully integrated native support is probably not going to happen as it is a lot of work and does not save any user time because they need to do the embedding manually anyway. Scripts are simple and very suitable for this problem.

**Comment by stopiccot** [comment]
But it would much more natural to add app bundle support to generated Xcode projects.

**Comment by jpakkane** [comment]
That is true. However it would require someone with proper XCode experience to create. I have roughly zero knowledge of it. This goes to the general XCode backend as well, which is, to be honest, a bit crappy ATM.
